core-concepts/token-bound-accounts

```diff
@@ -0,0 +1,46 @@
+---
+description: >-
+  An interface and registry that allows ERC-721 tokens to have their own smart contract accounts.
+---
+
+# Token Bound Accounts
+
+{% hint style="warning" %}
+The implementation of token bound accounts is work in progress. Hence, this material is subject to change.
+{% endhint %}
+
+## TL;DR
+
+Token Bound Accounts (TBAs) enhance the functionality of new and existing NFTs by equipping them with smart contract accounts. TBAs can be used to interact with decentralized applications (dApps), receive, store or transfer digital assets, either fungible or non-fungible. TBAs are backwards compatible with the ERC-721 standard, meaning existing non-fungible tokens (NFTs) can implement ERC-6551 without undergoing any fundamental changes.
+
+## Introduction
+
+Token Bound Accounts (TBAs) allow ERC-721 tokens to have their own smart contract accounts. This means that non-fungible tokens (NFTs) can own and interact with digital assets, either fungible or non-fungible, and interact with decentralized applications (dApps). There are several implementations of the TBA standard. When we refer to TBAs were are referring to the ERC-6551 standard implementation.
+
+## Token Bound Account Use Cases
+
+TBAs allow ERC-721 tokens to have their own smart contract account which opens a range of potential use cases: 
```

Comment:
What about ERC-1155?

Reply:
Updated to include ERC-1155 alongside all ERC-721 references.